docs/reference/public-api.md

```diff
@@ -0,0 +1,70 @@
+
+The public API contract is introduced with the 8.8 release.
+
+### Included in the public API
+
+The following components are officially part of the Camunda 8 public API:
+
+- The [orchestration cluster API](/apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview.md), which provides the primary interface for interacting with the orchestration cluster.
+- [Camunda Process Test](/apis-tools/testing/getting-started.md), which provides testing capabilities for process automation scenarios.
+- [Camunda Java client](/apis-tools/java-client/index.md), which provides Java libraries for interacting with Camunda 8.
+- [Camunda Spring Boot SDK](/apis-tools/spring-zeebe-sdk/getting-started.md), which provides Spring Boot integration for Camunda 8.
+
+## What is not included in the public API
+
+Some APIs are intentionally excluded from the public API definition. While we aim to keep them stable, they may evolve more rapidly and follow different lifecycle policies.
+
+Only the APIs and interfaces explicitly listed as ["Included in the public API"](#included-in-the-public-api) are covered by our SemVer stability guarantees. Any API or interface not explicitly listed is considered outside the public API contract. The [excluded APIs listed below](#excluded-apis) represent the most commonly used APIs to provide clarity, but this list is not exhaustive.
+
+### Alpha endpoints within the public API
+
+Even within APIs that are part of the public API contract, individual endpoints may be marked as [alpha features](/components/early-access/alpha/alpha-features.md). **Alpha endpoints are explicitly excluded from the public API stability guarantees**, even when they are part of an otherwise stable API.
+
+Alpha endpoints within the public API:
+
+- Are clearly marked as alpha in the API documentation
+- Can introduce breaking changes in minor or patch releases
+- Follow the [alpha feature policy](/components/early-access/alpha/alpha-features.md#alpha) rather than SemVer commitments
+- Are released for early testing and feedback before reaching general availability
+
+Always check the API documentation to identify alpha endpoints before building production integrations.
+
+### Excluded APIs
+
+The following APIs are **explicitly excluded** from the public API:
+
+- The [Web Modeler API](/apis-tools/web-modeler-api/index.md), used for browser-based model design.
+- The [Administration API](/apis-tools/administration-api/administration-api-reference.md), which enables administrative control and system configuration.
+- The [Optimize API](/apis-tools/optimize-api/overview.md), used for analytics, reporting, and performance insights.
+- The [Zeebe gRPC API](/apis-tools/zeebe-api/grpc.md), which provides low-level gRPC access to the workflow engine.
```

Comment:
❓ will we change this given https://camunda.slack.com/archives/C0PLU0Y12/p1751362178642819 ? CC @aleksander-dytko

Reply:
I removed this for now (not being explicit)